includes/admin/class-wcmp-export.php

```diff
@@ -1131,25 +1131,32 @@ public static function getItemWeight_kg($item, WC_Order $order): float
         }
 
         $weight      = (int) $product->get_weight();
+        $product_weight = WCMP_Export::calculatedKiloWeight($weight);
```

Comment:
De variabelen liever in camelcase. Ook zijn ze hier niet uitgelijnd

Reply:
Done